semantics/language/execution/expr/function-call.k

```diff
@@ -128,17 +127,13 @@ module C-EXPR-FUNCTION-CALL
           ) ...</call-stack>
           <control>
                <curr-tu> OldTu:K => Tu </curr-tu>
-               <curr-function> CurrFun:K </curr-function>
+               <curr-function> CurrFun:K => X </curr-function>
```

Comment:
why did you do this? the effect of this behavior is to make it so that the function name and the code location are out of sync if an error occurs between the application of this rule and the entry to the block. That is why I originally made this modification.